🤖 Auto-merge documentation from branches

buger opened the pull request:
### **User description**
## 📚 Documentation Merge Summary

This PR contains automatically merged documentation from multiple branches.

**Triggered by:**
- **Branch:** main
- **Commit:** b5314a081124afee5283ddb382e9699f19e6aa9b
- **PR:** #923 - [TT-15973] Charts RN 4.1.0.md

**Deployment Details:**
- **Generated from:** `branches-config.json`  
- **Run ID:** 19707994535
- **Subfolder:** `(root deployment)`
- **Timestamp:** $(date)

### Changes Include:
- ✅ Merged documentation from multiple branches
- ✅ Generated unified `docs.json` configuration
- ✅ Updated assets and content structure
- ✅ Cleaned up outdated version folders

---

🚦 **This PR will be processed by merge queue to ensure proper validation and ordering.**

Previous deployment PRs have been automatically closed to prevent conflicts.

[TT-15973]: https://tyktech.atlassian.net/browse/TT-15973?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ


___

### **PR Type**
Documentation


___

### **Description**
- Add Helm Charts 4.1.0 release notes

- Update overview to point to 4.1.0

- Include dependencies, downloads, changelog

- Clarify no breaking or deprecations


___

### Diagram Walkthrough


```mermaid
flowchart LR
  notes["Add 4.1.0 section in helm-chart.mdx"]
  deps["Dependencies, downloads, changelog added"]
  compat["Align with Tyk 5.8.8 LTS"]
  features["Tolerations + Topology Spread support"]
  overview["Update overview.mdx latest to 4.1.0"]
  notes -- "new release entry" --> deps
  deps -- "documents" --> features
  notes -- "mentions" --> compat
  overview -- "links and label updated" --> notes
```



<details> <summary><h3> File Walkthrough</h3></summary>

<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>helm-chart.mdx</strong><dd><code>Introduce Helm Charts 4.1.0 release notes</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

nightly/developer-support/release-notes/helm-chart.mdx

<ul><li>Add 4.1.0 release notes section.<br> <li> Document compatibility with Tyk 5.8.8 LTS.<br> <li> Provide dependencies, upgrade, downloads.<br> <li> Add changelog: tolerations and topology spread.</ul>


</details>


  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1054/files#diff-50d69597034e1c0ea048d11fea6ace9f2311b25fb7e7862479721802fbdba30d">+68/-0</a>&nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>overview.mdx</strong><dd><code>Refresh overview to reference 4.1.0</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

nightly/developer-support/release-notes/overview.mdx

<ul><li>Update latest Helm Charts to 4.1.0.<br> <li> Adjust latestData name and tag URL.<br> <li> Keep release date unchanged.</ul>


</details>


  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1054/files#diff-fb8ca7bde79e266244d0bcf78f942b283149eefc86d9437a7ea05f28c48c930d">+3/-3</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

</details>

___

marksou commented:
Hi @rewsmith regarding MongoDB and CE. I spoke to @excieve about this, and although the CE doesn't use Mongo, the Pump will not work without it, so it was felt it should be mentioned. Will edit the Helm info. A licence is not required for the CE version.

buger committed:
🤖 Auto-merge documentation from branches

Generated from branches-config.json at

- Merged documentation from multiple branches
- Generated unified docs.json
- Updated assets and content structure
